partcad-cli/src/partcad_cli/click/commands/export.py

```
@@ -0,0 +1,125 @@
+    help="The object is an assembly",
+    is_flag=True,
+)
+@click.option(
+    "-S",
+    "--scene",
+    help="The object is a scene",
+    is_flag=True,
+)
+@click.argument("object", type=str, required=False)  # Part (default), assembly or scene to test
+@click.pass_obj
+def cli(ctx, create_dirs, output_dir, format, package, recursive, sketch, interface, assembly, scene, object):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)